robottelo/cli/contentview.py

```diff
@@ -113,6 +114,12 @@ def publish(cls, options, timeout=1500000):
         cls.command_sub = 'publish'
         return cls.execute(cls._construct_command(options), ignore_stderr=True, timeout=timeout)
 
+    @classmethod
+    def purge(cls, options, timeout=1500000):
```

Comment:
```suggestion
    def purge(cls, options, timeout='25m'):
```
Would be nicer (in whole module).